torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -3557,6 +3557,19 @@ at::Tensor& AtenXlaType::zero_(at::Tensor& self) {
   return self;
 }
 
+at::Scalar AtenXlaType::_local_scalar_dense(const at::Tensor& self) {
+  static bool sync =
```

Comment:
Since we decided to put this option under experimental, could you reuse the `XLA_EXPERIMENTAL` and  use `ExperimentEnabled` to check whether `early_sync` is enabled. You can check the example in https://github.com/pytorch/xla/blob/d5b9bc143636370eaf3b2ecf8d8bbe738c904bc7/torch_xla/csrc/aten_xla_type.cpp#L1977